art/classifiers/ensemble.py

```diff
@@ -53,15 +53,20 @@ def __init__(self, classifiers, classifier_weights=None, channel_index=3, clip_v
                features. If arrays are provided, each value will be considered the bound for a feature, thus
                the shape of clip values needs to match the total number of features.
         :type clip_values: `tuple`
-        :param defences: Defences to be activated with the classifier.
-        :type defences: `str` or `list(str)`
+        :param preprocessing_defences: Preprocessing defence(s) to be activated with the classifier. Not applicable
+               in this classifier.
+        :type preprocessing_defences: :class:`.Preprocessor` or `list(Preprocessor)` instances
+        :param postprocessing_defences: Postprocessing defence(s) to be activated with the classifier.
+        :type postprocessing_defences: :class:`.Postprocessor` or `list(Postprocessor)` instances
         :param preprocessing: Tuple of the form `(subtractor, divider)` of floats or `np.ndarray` of values to be
                used for data preprocessing. The first value will be subtracted from the input. The input will then
-               be divided by the second one.
+               be divided by the second one. Not applicable in this classifier.
```

Comment:
Should we raise and exception if preprocessing is not None?

Reply:
Solved.